docs-md/tutorials/materialized.md

```diff
@@ -0,0 +1,367 @@
+    'table.whitelist' = 'call-center.calls',
+    'include.schema.changes' = 'false'
+);
+```
+
+After a few seconds, it should create a topic named `call-center-db.call-center.calls`. Print the raw topic contents to make sure it captured the initial rows that we seeded the calls table with:
+
+```sql
+PRINT 'call-center-db.call-center.calls' FROM BEGINNING;
+```
+
+If nothing prints out, the connector probably failed to launch. You can check ksqlDB's logs with:
+
+```
+docker logs -f ksqldb-server
+```
+
+You can also show the status of the connector in the ksqlDB CLI with:
+
+```
+DESCRIBE CONNECTOR calls_reader;
+```
+
+### Create the ksqlDB calls stream
+
+For ksqlDB to be able to use the topic that Debezium created, we need to declare a stream over it. Because we configured Kafka Connect with Schema Registry, we don't need to declare the schema of the data for the streams. It is simply inferred from the schema that Debezium writes with. Run the following at the ksqlDB CLI:
+
+```sql
+CREATE STREAM calls WITH (
+    kafka_topic = 'call-center-db.call-center.calls',
+    value_format = 'avro'
+);
+```
+
+### Create the materialized views
+
+A pretty common situation in call centers is the need to know what the current caller has called about in the past. Let's create a simple materialized view that keeps track of the distinct number of reasons that a user called for, and what the last reason was that they called for, too. This gives us an idea of how many kinds of inquiries the caller has raised, and also gives us context based on the last time they called.
+
+We do this by declaring a table called `support_view`. Keeping track of the distinct number of reasons a caller raised is as simple as grouping by the user name, then aggregating with `count_distinct` over the `reason` value. Similarly, we can retain the last reason the person called for with the `latest_by_offset` aggregation.
+
+Notice that Debezium writes events to the topic in the form of a map with "before" and "after" keys to make it clear what changed in each operation. That is why each column uses arrow syntax to drill into the nested `after` key.
+
+
+Run this statement at the prompt:
+
+```sql
+CREATE TABLE support_view AS
+    SELECT after->name AS name,
+           count_distinct(after->reason) AS distinct_reasons,
+           latest_by_offset(after->reason) AS last_reason
+    FROM calls
+    GROUP BY after->name
+    EMIT CHANGES;
+```
+
+We have our first materialized view in place. Let's create one more.
+
+It's useful to have an idea of the lifetime behavior of each caller. Rather than issuing a query over all the data every time we have a question about a caller, a materialized view makes it easy to incrementally update the answer as new information arrives over time. In this materialized view, we count the total number of times each person has called. We also compute the total number of minutes we've spent on the phone with this person. Run the following:
```

Comment:
```suggestion
It's useful to have an idea of the lifetime behavior of each caller. Rather than issuing a query over all the data every time there is a question about a caller, a materialized view makes it easy to update the answer incrementally as new information arrives over time. The following materialized view counts the total number of times each person has called and computes the total number of minutes spent on the phone with this person.
```